PhotoBooth.BL/Facades/OrderFacade.cs

```diff
@@ -0,0 +1,98 @@
+﻿using PhotoBooth.BL.Models.Item.Product;
+using PhotoBooth.BL.Models.Item.RentalItem;
+using PhotoBooth.BL.Models.Order;
+using PhotoBooth.BL.Queries;
+using PhotoBooth.DAL.Entity;
+using PhotoBooth.DAL.UnitOfWorkProviderModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace PhotoBooth.BL.Facades
+{
+    public class OrderFacade : FacadeBase<Order>, IOrderFacade
+    {
+        public OrderFacade(UnitOfWorkProvider provider) : base(provider) { }
+        public void CancelOrder(Guid orderId)
+        {
+            using (var uow = provider.GetUinOfWork())
+            {
+                uow.GetRepo<Order>().Delete(orderId);
+                uow.Save();
+            }
+        }
+
+        public OrderSummaryModel ChangeOrderPrice(Guid id, double newPrice)
+        {
+            throw new NotImplementedException("I don't know how to implement function");// TODO I'm not sure what to do here 
+        }
+
+        public void ConfirmOrder(Guid orderId)
+        {
+            throw new NotImplementedException("I don't know how to implement function");// TODO I'm not sure what to do here 
+        }
+
+        public ICollection<OrderListModel> GetAllOrders()
+        {
+            return new OrderListQuery(provider).ExecuteAsync();
+        }
+
+        public ICollection<OrderListModel> GetOrdersByUser(Guid userId)
+        {
+            throw new NotImplementedException("Team consult needed");// TODO I'm not sure what to do here 
```

Comment:
If the problem is ApplicationUser having string id instead of guid, than you can cast string to guid `var guidId=Guid.Parse(id);` or modify the ApplicationUser to use guid via generic argument